xonsh/timings.py

```diff
@@ -46,10 +46,10 @@ def clocku():
             """clocku() -> floating point number
             Return the *USER* CPU time in seconds since the start of the process.
             This is done via a call to resource.getrusage, so it avoids the
-            wraparound problems in time.clock()."""
+            wraparound problems in time.perf_counter()."""
```

Comment:
I'm not sure to understand what are the «wraparound problems» there, so I don't know if a deeper refactoring should be done, or if the comment still make sense after changing the call to `time.perf_counter()`.